src/k8s-engine/wsl.ts

```diff
@@ -1103,16 +1104,41 @@ export default class WSLBackend extends events.EventEmitter implements K8s.Kuber
         })(),
         ];
 
+        await Promise.all(downloadingActions);
```

Comment:
We should extract the downloading code to a helper so that we can still do it in parallel with the other stuff.
(Actually, why is the fallback code not part of `ensureK3sImages`?)